packages/components/src/Helpers.purs

```diff
@@ -117,3 +122,17 @@ getMinStartDate nextDelivery = do
           soonestStart = adjust soonestDuration (DateTime.date now)
           byNextIssue = max <$> soonestStart <*> nextDelivery
       pure $ byNextIssue <|> soonestStart
+
+matchesRegex :: String -> String -> Boolean
+matchesRegex regex str = isJust $ do
+  re <- hush $ Regex.regex regex mempty
+  Regex.match re str
+
+-- This same deterministic function is in Lettera's code
+editorialIdToUuid :: String -> UUID.UUID
+editorialIdToUuid editorialId =
+  UUID.genv5UUID ("https://hblmedia.fi/" <> editorialId) url_namespace
+
+-- Constant defined in the UUIDv5 standard
+url_namespace :: UUID.UUID
+url_namespace = unsafePartial $ fromJust $ UUID.parseUUID "6ba7b811-9dad-11d1-80b4-00c04fd430c8"
```

Comment:
Please set your editor to add line feed after the final line.

Reply:
Fixed